### PR TITLE
fix folder structure and build targets

### PR DESCRIPTION
**Pull request by 3stadt**
I moved the content of the SRC folder into the root and adjusted the package name.
That makes for a more idiomatic approach, also potentially things are now go-gettable.

### CODE REVIEW

**Comment by 3stadt** [comment]
Also, makefile had some unused build targets and was missing others, fixed that.

File: makefile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+# ----------------------------------------------------------------------------------------------------------------
+
+install: ## Installs all dependencies
+	go mod download
+
+build: ## Builds the binary
+	env GOOS=linux GOARCH=amd64 go build -o ./build/linux/amd64/dockware-cli .
```
Comment (shyim) [inline makefile:18]: you could use also goreleaser 🤔

Comment (3stadt) [reply makefile:18]: I though about that, but as long as boxblinkracer is just releasing a binary I'd say the task is too simple to warrant another dependency, and make is pretty much universally available.

Comment (boxblinkracer) [reply makefile:18]: hehe, totally open for that,
at the moment theres just a small automation in the background to release new binaries on the website (didnt have more time)
but we can totally change and improve it :)

**Pull request merged**